extra-creativity/darktable/autobuild/prepare

```diff
@@ -2,3 +2,6 @@ if [[ "${CROSS:-$ARCH}" = "ppc64el" ]]; then
     abinfo "Append -DNO_WARN_X86_INTRINSICS to fix build ..."
     export CFLAGS="${CFLAGS} -DNO_WARN_X86_INTRINSICS"
 fi
+
+abinfo "Copying zh_CN translation"
+cp -v "${SRCDIR}/autobuild/zh_CN.po" "${SRCDIR}/po/zh_CN.po"
```

Comment:
You should probably maintain this in https://github.com/AOSC-Dev/translations and pull it from the `spec` instead.

Reply:
Will do.